vendor/bat-native-ads/src/bat/ads/confirmation_type.cc

```diff
@@ -11,6 +11,9 @@ static const char kConfirmationTypeClick[] = "click";
 static const char kConfirmationTypeDismiss[] = "dismiss";
 static const char kConfirmationTypeView[] = "view";
 static const char kConfirmationTypeLanded[] = "landed";
+static const char kConfirmationTypeFlag[] = "flag";
```

Comment:
@tmancey These are the new confirmation types, but note that they wouldn't ever be user-visible (unlike view, dismiss, etc.)